app/components/attachment_button.js

```diff
@@ -382,6 +398,25 @@ export default class AttachmentButton extends PureComponent {
             icon: 'file',
         });
 
+        // Show option to remove image in edit profile screen
+        // default image will not include query string with timestamp
+        if (this.props.removeProfileImage && this.isCustomProfileImage()) {
+            options.items.push({
+                action: () => this.removeProfileImage(),
```

Comment:
Can you just do action: this.removeProfileImage without the arrow function?

Reply:
sure, originally i thought i would need a param there.

Reply:
not really you should do

```js
options.items.push({
    action: this.removeProfileImage,
   ...
});
```

Reply:
Makes sense - since this is just a reference and the binding happens on the lower level component.